web/consumers.py

```
@@ -0,0 +1,207 @@
+        """Send current participants list to the group"""
+        try:
+            participants = await self.get_participants_list()
+            await self.channel_layer.group_send(
+                self.room_group_name,
+                {
+                    'type': 'participants_list',
+                    'participants': participants
+                }
+            )
+        except Exception as e:
+            logger.error(f"Error sending participants list: {str(e)}") 
```

Ruff (0.11.9)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)